__tests__/app.test.js

```diff
@@ -113,10 +113,95 @@ describe("GET /api/reviews/:review_id", () => {
   });
 });
 
+describe("GET /api/reviews", () => {
+  it("returns 200 with an array of review objects", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((result) => {
+        return result.body.reviews;
+      })
+      .then((reviews) => {
+        expect(Array.isArray(reviews)).toBe(true);
+      });
+  });
+  it("reviews has required properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .then((result) => {
+        return result.body.reviews;
+      })
+      .then((reviews) => {
+        reviews.forEach((review) => {
+          expect(review).toHaveProperty("owner");
+          expect(review).toHaveProperty("title");
+          expect(review).toHaveProperty("review_id");
+          expect(review).toHaveProperty("category");
+          expect(review).toHaveProperty("review_img_url");
+          expect(review).toHaveProperty("created_at");
+          expect(review).toHaveProperty("votes");
+          expect(review).toHaveProperty("designer");
+        });
+      });
+  });
+  it("has property comment_count which is the number of comments with this review_id", () => {
+    return request(app)
+      .get("/api/reviews")
+      .then((result) => {
+        return result.body.reviews;
+      })
+      .then((reviews) => {
+        return db
+          .query(`SELECT * FROM comments;`)
+          .then((result) => result.rows)
+          .then((comments) => {
+            reviews.forEach((review) => {
+              //Calculate comments for each review
+              let commentCount = 0;
```

Comment:
This is a very complicated test and It carries on from the logic you have in your model which I have left a comment for. Please refactor it using what I suggested and the test will also be a lot simpler. Please also test to ensure that the comment_count is a number as it will be a string by default